Sources/Transporter.swift

```diff
@@ -48,7 +48,7 @@ final class NetServiceTransport: NSObject {
     // For some reason, Stream Task could send a big file
     // https://github.com/ProxymanApp/atlantis/issues/57
     static let MaximumSizePackage = 52428800 // 50Mb
-
+    static let MaximumSizeResponse = 2097152 // 2MB
```

Comment:
I suppose that we should increase the MaximumSizeResponse to 5Mb or 10Mb.

Reply:
Yeah, It likely too small, I'm fixing to 10 MB and you can check it.
Thank you